release-calendar/src/server/app.ts

```diff
@@ -68,6 +72,39 @@ async function main(): Promise<void> {
     app.get('/', async (req, res) => {
       res.sendFile(path.join(DIST_DIR, 'index.html'));
     });
+
+    app.post('/insert/', async (req, res) => {
+      // authorization
+      const authorization = req.header('authorization');
+      const auth = authorization.split('Basic ')[1];
```

Comment:
Something I thought about after seeing `bearer xxxxx` in the other auth refactor: this should probably be case insensitive for resiliency

```suggestion
      const auth = authorization.split(/Basic /i)[1];
```

Reply:
Nice!